src/js/actions/transform.js

```diff
@@ -390,9 +433,9 @@ define(function (require, exports) {
                 positions: []
             },
             layerOneActions = _getMoveLayerActions
-                .call(this, document, layers.get(0), newPositions.get(0), payload.positions),
+                .call(this, document, layers.get(0), newPositions.get(0), undefined, payload.positions),
```

Comment:
`this` is not in use in the helper function, so using `call` is not necessary? Please search for `_getMoveLayerActions`